bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -536,6 +675,478 @@ pub struct ReferenceUnlockBlockDto {
+            FeatureBlock::Issuer(v) => Self::Issuer(IssuerFeatureBlockDto(v.address().into())),
+            FeatureBlock::DustDepositReturn(v) => Self::DustDepositReturn(DustDepositReturnFeatureBlockDto(v.amount())),
+            FeatureBlock::TimelockMilestoneIndex(v) => {
+                Self::TimelockMilestoneIndex(TimelockMilestoneIndexFeatureBlockDto(v.index()))
+            }
+            FeatureBlock::TimelockUnix(v) => Self::TimelockUnix(TimelockUnixFeatureBlockDto(v.timestamp())),
+            FeatureBlock::ExpirationMilestoneIndex(v) => {
+                Self::ExpirationMilestoneIndex(ExpirationMilestoneIndexFeatureBlockDto(v.index()))
+            }
+            FeatureBlock::ExpirationUnix(v) => Self::ExpirationUnix(ExpirationUnixFeatureBlockDto(v.timestamp())),
+            FeatureBlock::Indexation(v) => Self::Indexation(IndexationFeatureBlockDto(v.to_string())),
+            FeatureBlock::Metadata(v) => Self::Metadata(MetadataFeatureBlockDto(v.to_string())),
+        }
+    }
+}
+
+impl TryFrom<&FeatureBlockDto> for FeatureBlock {
+    type Error = Error;
+
+    fn try_from(value: &FeatureBlockDto) -> Result<Self, Self::Error> {
+        Ok(match value {
+            FeatureBlockDto::Sender(v) => Self::Sender(SenderFeatureBlock::new((&v.0).try_into()?)),
+            FeatureBlockDto::Issuer(v) => Self::Issuer(IssuerFeatureBlock::new((&v.0).try_into()?)),
+            FeatureBlockDto::DustDepositReturn(v) => Self::DustDepositReturn(DustDepositReturnFeatureBlock::new(v.0)?),
+            FeatureBlockDto::TimelockMilestoneIndex(v) => {
+                Self::TimelockMilestoneIndex(TimelockMilestoneIndexFeatureBlock::new(v.0))
+            }
+            FeatureBlockDto::TimelockUnix(v) => Self::TimelockUnix(TimelockUnixFeatureBlock::new(v.0)),
+            FeatureBlockDto::ExpirationMilestoneIndex(v) => {
+                Self::ExpirationMilestoneIndex(ExpirationMilestoneIndexFeatureBlock::new(v.0))
+            }
+            FeatureBlockDto::ExpirationUnix(v) => Self::ExpirationUnix(ExpirationUnixFeatureBlock::new(v.0)),
+            FeatureBlockDto::Indexation(v) => Self::Indexation(IndexationFeatureBlock::new(
+                hex::decode(&v.0).map_err(|_e| Error::InvalidSemanticField("IndexationFeatureBlock"))?,
+            )?),
+            FeatureBlockDto::Metadata(v) => Self::Metadata(MetadataFeatureBlock::new(
+                hex::decode(&v.0).map_err(|_e| Error::InvalidSemanticField("MetadataFeatureBlock"))?,
+            )?),
+        })
+    }
+}
+
+/// Describes an alias account in the ledger that can be controlled by the state and governance controllers.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct AliasOutputDto {
+    #[serde(rename = "type")]
+    pub kind: u8,
+    // Amount of IOTA tokens held by the output.
+    pub amount: u64,
+    // Native tokens held by the output.
+    pub native_tokens: Vec<NativeTokenDto>,
```

Comment:
Don't we need `#[serde(rename = "nativeTokens")]` here?

Reply:
Also for the other fields and same for FoundryOutputDto and NftOutputDto

Reply:
you're right, fixing...

Reply:
I have renamed the fields. By the way are you aware of a spec that specifies how the JSON fields need to be named?

Reply:
No